view/components/filter-bar/select-date-from.js

```diff
@@ -8,8 +8,8 @@ module.exports = function (/* opts */) {
 	name = opts.name || 'dateFrom';
 	return input({
 		id: opts.id || 'date-from-input',
-		type: 'date',
-		name: name,
+		type: opts.type || 'date',
+		name: opts.name || name,
```

Comment:
Just saw this one, it is redundant. In line 8 you already use opts.name to setup name.